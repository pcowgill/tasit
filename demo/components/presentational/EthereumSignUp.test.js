import React from "react";
import { shallow } from "enzyme";
import EthereumSignUp from "./EthereumSignUp";

describe("EthereumSignUp", () => {
  it("renders the component", async () => {
    expect(shallow(<EthereumSignUp />)).toMatchSnapshot();
  });
});
