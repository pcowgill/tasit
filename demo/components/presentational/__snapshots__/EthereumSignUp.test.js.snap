// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EthereumSignUp renders the component 1`] = `
<KeyboardAvoidingView
  behavior="padding"
  enabled={true}
  keyboardVerticalOffset={104.02}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <LargeText>
    Cool. Let's start by picking your Tasit username.
  </LargeText>
  <EthereumSignUpForm />
</KeyboardAvoidingView>
`;
